setup.py

```diff
@@ -306,9 +307,28 @@ def find_version(filepath: str) -> str:
         raise RuntimeError("Unable to find version string.")
 
 
+# Neuralmagic packaging ENV's
```

Comment:
could these be added to `nm-vllm/vllm/envs.py`?

This is the central location for env vars - perhaps not b/c this is setup.py, but just thought I would ask

Reply:
i can take a look